react-native-ssi/src/types.ts

```diff
@@ -96,3 +96,187 @@ export type TermsOfUse = {
 export type Proof = {
   type: string | string[];
 };
+
+export type CredentialManifest = {
+  id: string;
+  spec_version: string;
+  issuer: CredentialIssuer;
+  output_descriptors: OutputDescriptor[];
+  name?: string;
+  description?: string;
+  format?: ClaimFormatDesignation;
+  presentation_definition?: PresentationDefinition;
+};
+
+export type CredentialIssuer = {
+  id: string;
+  name?: string;
+  styles?: EntityStyle;
+  comment?: string;
+};
+
+export type EntityStyleImage = {
```

Comment:
I think the `Entity` gobbledygook can be removed

Reply:
Maybe? It's implementing this spec: https://identity.foundation/wallet-rendering/v0.0.1/#entity-styles

Thinking we can just get rid of it altogether?

Reply:
Oh ignore what I said